src/storage/data_table.cpp

```diff
@@ -43,6 +45,7 @@ void DataTable::Select(transaction::TransactionContext *const txn, const TupleSl
     StorageUtil::ApplyDelta(accessor_.GetBlockLayout(), *(version_ptr->Delta()), out_buffer);
     version_ptr = version_ptr->Next();
   }
+  data_table_counter_.IncNumSelect();
```

Comment:
nit: don't use abbreviation.

Second thought: does it make sense to add something not 1 in a call to Increment? Maybe the right way is to take in an `increment_by` argument that defaults to 1 (don't tell Andy)

Reply:
Don't do the default argument :)